helper-scripts/ansible/roles/mailcow-dockerized/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+- name: Make sure git is installed
+  package:
+    name: git
+    state: present
+
+- name: Clone the master branch of mailcow-dockerized repo
+  git:
+    repo: https://github.com/mailcow/mailcow-dockerized
+    dest: /opt/mailcow-dockerized
+    umask: "0022"
+    #force: yes # overwrite modified files
+  ignore_errors: yes
+
+- name: Add mail.* on ADDITIONAL_SAN
+  replace:
+    path: /opt/mailcow-dockerized/generate_config.sh
+    regexp: "^ADDITIONAL_SAN.*"
+    replace: "ADDITIONAL_SAN=mail.*"
+
+- name: Generate a configuration file
+  shell: ./generate_config.sh
```

Comment:
You need to add:
```
  args:
    executable: /bin/bash
```
since the generate_config.sh is a bash script and the default shell of the ansible shell module is sh.

Reply:
Yes but the script has "#!/usr/bin/env bash" at the top so bash is used to run it